pkg/yurthub/util/util.go

```diff
@@ -86,13 +86,15 @@ const (
 	CacheUserAgentsKey    = "cache_agents"
 	PoolScopeResourcesKey = "pool_scope_resources"
 
+	MultiplexerProxyClientUserAgent = "multiplexer-proxy"
```

Comment:
how about name `MultiplexerProxyClientUserAgent ` to fmt.Sprintf("multiplexer-proxy-%s", strings.ToLower(nodeName))?

Reply:
The certificate contains node information. I suggest not adding the node name.